en/docs/develop-components/develop-integrations/develop-a-manual-task.md

```diff
@@ -1,106 +1,97 @@
-          | **Dependencies.toml** | Lists the dependencies required for the project                      |
-          | **main.bal**          | Contains the entry point of the project, including the main function |
-          | **types.bal**         | Contains custom data types used in the project                       |
-          | **utils.bal**         | Contains utility functions and helper functions used in the project  |
-          
- - Go to [OpenWeatherMap](https://openweathermap.org/) and sign up to obtain an API key. For details on how to obtain an API key, see the [OpenWeatherMap documentation](https://openweathermap.org/appid#signup).
+2. **Fork the Repository**: Fork the [Choreo samples repository](https://github.com/wso2/choreo-samples), which contains the sample integration for this guide.
 
+    !!! info "Repository File Structure"
+        The sample code for this guide is in the `<sample-repository-dir>/weather-to-email-integration` directory. The following table describes the key files in the repository:
+
+        | **File Path**         | **File Content**                                                     |
+        | --------------------- | -------------------------------------------------------------------- |
+        | **Ballerina.toml**    | Contains metadata about the project                                  |
+        | **Dependencies.toml** | Lists the dependencies required for the project                      |
+        | **main.bal**          | Contains the entry point of the project, including the main function |
+        | **types.bal**         | Contains custom data types used in the project                       |
+        | **utils.bal**         | Contains utility functions and helper functions used in the project  |
+
+3. **Obtain an OpenWeatherMap API Key**: Go to [OpenWeatherMap](https://openweathermap.org/) and sign up to obtain an API key. For details, see the [OpenWeatherMap documentation](https://openweathermap.org/appid#signup).
 
 ## Step 1: Create a manual task component
 
-1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page. 
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page.
 2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
 3. Click the **Manual Task** card.
-4. Enter a unique name and a description for the component. You can use the name and description given below:
+4. Enter a unique name and description for the component. You can use the following values:
 
     | **Field**          | **Value**                        |
     | ------------------ | -------------------------------- |
-    | **Component Name** | `WeatherToEmail`                 |
-    | **Description**    | `My first manual task`           |
+    | **Component Name** | WeatherToEmail                   |
```

Comment:
We generally use code font for user input values. See https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#text-formatting-guidelines and https://developers.google.com/style/text-formatting